packages/react/src/progress-indicator/ProgressIndicatorItemLink.tsx

```diff
@@ -0,0 +1,47 @@
+import { LinkProps } from '../core';
+import { TextLink } from '../text-link';
+import { ProgressIndicatorItem } from './ProgressIndicatorItem';
+import {
+	ProgressIndicatorBackground,
+	ProgressIndicatorItemStatus,
+} from './utils';
+
+export type ProgressIndicatorItemLinkProps = LinkProps & {
+	/** Determines what style the item displays as.
+	 *
+	 * `blocked` = This step is not yet available for the user to start.
+	 *
+	 * `doing` = **(deprecated)** This step has been started and the user is actively working on this step (Use `started` with `isActive: true` instead).
```

Comment:
We shouldn't be recommending "(Use `started` with `isActive: true` instead)."

Reply:
Why not?
We say "don't use this" but if they are already using it, the "(Use started with isActive: true instead)" bit guides the developer to correct their outdated code.

Reply:
Ahh I missed this. `isActive` is deprecated, I'll remove.

- [ ] remove (Use started with isActive: true instead).